Bottom: 9e174670838c5fe248ae945717232b2b538f4cd7
Top:    fd0c525c655f2908b30757c7eb487260282c5d88
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-01 20:37:29 -0400

Refresh of device-trees

---

diff --git a/arch/arm/mach-lpc31xx/board-ea313x.c b/arch/arm/mach-lpc31xx/board-ea313x.c
index fc51b32..76679ce 100644
--- a/arch/arm/mach-lpc31xx/board-ea313x.c
+++ b/arch/arm/mach-lpc31xx/board-ea313x.c
@@ -50,18 +50,6 @@
 #include <mach/system.h>
 #include <mach/dt.h>
 
-#ifdef CONFIG_OF
-
-static void __init ea_add_device_ssd1289(void)
-{
-	MPMC_STCONFIG0 = 0x81;
-	MPMC_STWTWEN0  = 0;
-	MPMC_STWTOEN0  = 0;
-	MPMC_STWTRD0   = 31;
-	MPMC_STWTPG0   = 0;
-	MPMC_STWTWR0   = 3;
-	MPMC_STWTTURN0 = 0;
-}
 
 /*
  * DM9000 ethernet device
@@ -109,26 +97,6 @@ static struct dm9000_plat_data dm9000_platdata = {
 	.inblk = dm9000_inblk,
 };
 
-static void __init ea_add_device_dm9000(void)
-{
-	/*
-	 * Configure Chip-Select 2 on SMC for the DM9000.
-	 * Note: These timings were calculated for MASTER_CLOCK = 90000000
-	 *  according to the DM9000 timings.
-	 */
-	MPMC_STCONFIG1 = 0x81;
-	MPMC_STWTWEN1 = 1;
-	MPMC_STWTOEN1 = 1;
-	MPMC_STWTRD1 = 4;
-	MPMC_STWTPG1 = 1;
-	MPMC_STWTWR1 = 1;
-	MPMC_STWTTURN1 = 2;
-	/* enable oe toggle between consec reads */
-	SYS_MPMC_WTD_DEL1 = 0x24;
-
-}
-
-
 
 /* spi_board_info.controller_data for SPI slave devices,
  * copied to spi_device.platform_data ... mostly for dma tuning
@@ -180,7 +148,7 @@ static int __init lpc313x_spidev_register(void)
 
 	return spi_register_board_info(info, 3);
 }
-arch_initcall(lpc313x_spidev_register);
+//arch_initcall(lpc313x_spidev_register);
 #endif
 
 
@@ -207,8 +175,6 @@ struct of_dev_auxdata ea3131_auxdata_lookup[] __initdata = {
 static void __init ea3131_dt_init(void)
 {
 	lpc31xx_dt_init_common(ea3131_auxdata_lookup);
-	ea_add_device_ssd1289();
-	ea_add_device_dm9000();
 }
 
 #if defined(CONFIG_USB_EHCI_HCD)
@@ -220,7 +186,7 @@ static int __init ea_usb_power(void)
 	//ret = gpio_direction_output(VBUS_PWR_EN, 1);
 	return ret;
 }
-late_initcall(ea_usb_power);
+//late_initcall(ea_usb_power);
 #endif
 
 static const char *ea3131_dt_match[] __initconst = {
@@ -237,5 +203,5 @@ DT_MACHINE_START(EA313X, "NXP EA3131 (Device Tree Support)")
 	.dt_compat	= ea3131_dt_match,
 	.restart	= arch_reset,
 MACHINE_END
-#endif
+
